react/Section.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+import { useResponsiveValue } from 'vtex.responsive-values'
+
+interface Props {
+  borderless?: boolean
+  marginBottom?: number
+  paddingBottom?: number
+  width?: string | { mobile: string; desktop: string }
```

Comment:
You could create this type

```typescript
interface ResponsiveInput<T> {
    mobile: T
    desktop: T
    tablet: T
}
type MaybeResponsiveInput<T> = ResponsiveInput<T> | T
```

```suggestion
  width?: MaybeResponsiveInput<string>
```